plugins/sfRadPlugin/modules/sfRadPlugin/templates/indexSuccess.php

```diff
@@ -86,7 +86,7 @@
     <div class="titleNotes">
       <ul>
         <?php foreach ($resource->getNotesByTaxonomy(['taxonomyId' => QubitTaxonomy::RAD_TITLE_NOTE_ID]) as $item) { ?>
-          <?php if (0 != count($item->getContent(['cultureFallback' => true]))) { ?>
+          <?php if (0 != strlen($item->getContent(['cultureFallback' => true]))) { ?>
```

Comment:
Likewise, this could be replaced with the equivalent:

```PHP
if (!empty($item->getContent(['cultureFallback' => true])) {
```

Which is easier to read I think.